honeybee_schema/radiance/properties.py

```diff
@@ -0,0 +1,173 @@
+"""Properties Schema"""
+from pydantic import Field, constr, validator, root_validator
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import Void, ModifierBase
+from .modifierset import ModifierSet
+
+
+class _PropertiesBaseAbridged(IDdBaseModel):
+    """Base class of Abridged Radiance Properties."""
+
+    modifier: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
+        description='A string for a Honeybee Radiance Modifier.'
+        )
+
+    modifier_blk: str = Field(
+        default=None,
+        min_length=1,
+        max_length=100,
+        description='A string for a Honeybee Radiance Modifier to be used '
+                    'in direct solar simulations and in isolation studies (assessing'
+                    'the contribution of individual objects).'
+        )
+
+
+class _PropertiesBase(IDdBaseModel):
```

Comment:
This class should be deleted as we never use non-abridged radiance properties in the Model schema.